src/simtools/simtel/simtel_config_writer.py

```diff
@@ -422,3 +439,81 @@ def _get_array_triggers_for_telescope_type(self, array_triggers, telescope_type)
             if trigger_dict["name"] == telescope_type + "_array":
                 return trigger_dict
         return None
+
+    def write_dummy_telescope_configuration_file(
+        self, parameters, config_file_path, telescope_name
+    ):
+        """
+        Write 'dummy' telescope configuration file used as zeroth telescope in sim_telarray.
+
+        Replaces key telescope configuration values with dummy values.
+
+        Parameters
+        ----------
+        parameters: dict
+            Telescope parameters used as template.
+        config_file_path: str or Path
+            Path of the dummy configuration file to write on.
+        telescope_name: str
+            Name of the telescope.
+        """
+        self._logger.debug(f"Writing {telescope_name} telescope config file {config_file_path}")
+        defaults = {
```

Comment:
Weird to call it defaults here, I suggest either invalid or dummy values, depending on which of the names you decide to stick with.

Reply:
Changed to `dummy_defaults`.